lib/builderator/tasks/packer.rb

```diff
@@ -101,6 +101,7 @@ def wait(profile)
               state = regional_image.nil? ? 'unknown' : regional_image.state
               image_id = regional_image.nil? ? 'unknown' : regional_image.image_id
 
+              puts "ERROR: Unknown image state, region: #{region}, image_name: #{image_name}, image_details: #{regional_image}" if regional_image.nil?
```

Comment:
if `regional_image.nil?` doesn't that make adding `image_details: #{region_image}` pointless?